docs(docs-infra): improve the content of attribute directives

shejialuo opened the pull request:
Improve docs "Understanding Angular-Directives-Attribute Directives"

In `highlight.directive.1.ts`, add the `private` keyword for `el` property
to make it correct and consistent with subsequent examples.

For section "Setting the value with user input", add a step to tell the
reader change the method `onMouseEnter`, thus more readable.

For section "Binding to a secon property", add a step to tell the reader
should add an alias because the docs doesn't mention it at all, thus
more readable.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

https://angular.io/guide/attribute-directives

Issue Number: N/A


## What is the new behavior?

In `highlight.directive.1.ts`, add the `private` keyword for `el` property to make it correct and consistent with subsequent examples.

After:

![image](https://user-images.githubusercontent.com/56911263/148189961-8c5fc602-cac2-4648-beb9-b74d5aa52e41.png)

For section "Setting the value with user input", add a step to tell the reader change the method `onMouseEnter`, thus more readable.

After:

![image](https://user-images.githubusercontent.com/56911263/148199848-ed2d5294-cd0a-45a5-8fce-a488bc27557e.png)


For section "Binding to a second property", add a step to tell the reader should add an alias because the docs doesn't mention it at all, thus more readable

After

![image](https://user-images.githubusercontent.com/56911263/148190231-a62c718f-69d2-4a60-9330-d1c6db22581f.png)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

atscott commented on aio/content/examples/attribute-directives/src/app/highlight.directive.ts:
```diff
@@ -11,7 +11,9 @@ export class HighlightDirective {
   @Input() defaultColor = '';
   // #enddocregion defaultColor
 
+  // #docregion highlightColor
```
I don't think the intent of this tutorial was to introduce aliases. Can we instead change the code and content to consistently use `appHighlight`?

shejialuo replied:
It's a good idea to consistently use `appHighlight`. And I have changed the PR.

shejialuo committed:
docs(docs-infra): improve the content of attribute directives

Improve docs "Understanding Angular-Directives-Attribute Directives"

In `highlight.directive.1.ts`, add the `private` keyword for `el` property
to make it correct and consistent with subsequent examples.

For section "Setting the value with user input", add a step to tell the
reader change the method `onMouseEnter`, thus more readable.

For section "Binding to a second property", delete the aliase for the
`Input` property, and changes the name back to `appHighlight` in the code
to make it consistent with the whole tutorial.

mary-poppins commented:
You can preview 89eb30a at https://pr44628-89eb30a.ngbuilds.io/.
You can preview 3c2d822 at https://pr44628-3c2d822.ngbuilds.io/.

atscott commented:
This PR was merged into the repository by commit 7f51f1dcae577662f917c7b1c9ce3f237c1a7a6d.
